app/brave_generated_resources.grd

```diff
@@ -1105,10 +1105,18 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_IMPORT_FROM_EDGE" desc="browser combo box: Microsoft Edge Legacy">
         Microsoft Edge Legacy
       </message>
-
       <message name="IDS_IMPORTED_FROM_BOOKMARK_FOLDER" desc="Name for bookmark panel folder imported from another browser">
         Imported from <ph name="BROWSER_NAME">$1<ex>Chrome</ex></ph>.
       </message>
+      <!-- Chromium only has these strings for non-Android, but we build the code that uses them on Android as well -->
```

Comment:
So isn't the issue here that we shouldn't be building it on android? Shouldn't we fix that instead of patching here? Or do we have the importer on android? If so, this comment is confusing

Reply:
We have the importer on Android (cc: @samartnik  to confirm). This is not a patch - this is our own strings file. I added these strings to our strings flag so that we don't have to patch the upstream one.

Reply:
Sure, I'll fix the comment